common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -24,6 +24,7 @@
   public FeatureFlagsConfig featureFlags;
   public BillingConfig billing;
   public ActionAuditConfig actionAudit;
+  public RDRConfig rdrServer;
```

Comment:
[Non-opt] Consistency note: please follow the pattern used for other fields, where the class name is "FooConfig" and the field name is "foo". I think the base name here could be "RdrExport", so you'd have

public RdrExportConfig rdrExport;